frontend/src/App.js

```diff
@@ -1,11 +1,21 @@
-//import {Switch, Route} from "react-router-dom";
+import { Switch, Route } from "react-router-dom";
 import ExplorePage from "./pages/ExplorePage";
 import styled from "styled-components/macro";
+import LoginPage from "./pages/LoginPage";
+import Header from "./components/Header";
 
 function App() {
   return (
     <Wrapper>
-      <ExplorePage />
```

Comment:
Du brauchst nach dem Wrapper "<Router>", ansonsten funktionieren Routes nicht.

Reply:
Danke dir, ich habe den Router bei mir in der index.js, somit ist er vor dem Switch, man sieht es nur nicht sofort :)